app/server/lib/GranularAccess.ts

```diff
@@ -2962,7 +2962,7 @@ export class CensorshipInfo {
       columnRefToColId.set(colRef, colId);
       if (uncensoredTables.has(tableRef)) { continue; }
       const tableId = tableRefToTableId.get(tableRef);
-      if (!tableId) { throw new Error('table not found'); }
+      if (!tableId) { throw new Error('table not found: ' + tableRef); }
```

Comment:
`tableRef` would typically be a number, and `tableId` a string. Are you confident you want `tableRef` here?

Reply:
I would say yes. `tableId` is empty when entering in this condition, and we only have the tableRef to show.

Maybe I could also print the `colId` for a more intelligible value to print?

Reply:
Oh, of course, that makes sense, my eyes skipped the conditional :) - this is fine.